packages/trackerless-network/src/logic/createRandomGraphNode.ts

```diff
@@ -29,6 +30,8 @@ const createConfigWithDefaults = (config: RandomGraphNodeConfig): StrictRandomGr
     const nearbyContactPool = config.nearbyContactPool ?? new PeerList(peerId, numOfTargetNeighbors + 1)
     const randomContactPool = config.randomContactPool ?? new PeerList(peerId, maxNumberOfContacts)
     const targetNeighbors = config.targetNeighbors ?? new PeerList(peerId, maxNumberOfContacts)
+    const inspectingConnections = config.inspectingConnections ?? new PeerList(peerId, 10)
```

Comment:
E.g.
```suggestion
    const inspectingNodes = config.inspectingNodes ?? new PeerList(peerId, 10)
```

As peer list is a list of nodes, not connections?